### DISCUSSION TITLE
Atom Not Opening on Centos 7

### DISCUSSION THREAD

**Discussion by wright-james** [atom/atom#22620] [General]
It looks like it's going to open then nothing except I can see processes have been kicked off.

I've found a couple of convoluted solutions that suggest installing anaconda or something to get a different version of some dependency software.  That can't be the fix.  Can it?

Thanks in Advance,

Jim W

**Comment by DeeDeeG** [comment]
Centos 7 is quite old now (first released in 2014! Though I understand someone is still maintaining it...). I think, due to the need for updated language features, newer C standard library and so on, running the latest Atom on Centos 7 would be hard, and as far as I know isn't supported.

I suppose yes, there might be a way to install newer stuff on Centos 7 and it might work. I'm not deeply familiar with Red Hat Enterprise Linux, Fedora or CentOS... If you can possibly move to a newer OS, that should let you use Atom.

**Reply by wright-james** [reply]
Change is bad!!  You're right though, Ima bite the bullet today.

Peace and Gratitude!
